--- pulseview-a7553da/pv/sigsession.cpp~	2013-03-09 16:09:28.000000000 +0100
+++ pulseview-a7553da/pv/sigsession.cpp	2013-03-09 16:45:26.008835730 +0100
@@ -172,7 +172,7 @@
 void SigSession::feed_in_header(const sr_dev_inst *sdi)
 {
 	shared_ptr<view::Signal> signal;
-	uint64_t *sample_rate;
+	uint64_t *sample_rate = NULL;
 	unsigned int logic_probe_count = 0;
 	unsigned int analog_probe_count = 0;
 
--- pulseview-a7553da/pv/view/viewport.cpp~	2013-03-09 16:09:28.000000000 +0100
+++ pulseview-a7553da/pv/view/viewport.cpp	2013-03-09 17:23:03.369361232 +0100
@@ -104,11 +104,6 @@
 	}
 }
 
-void Viewport::mouseReleaseEvent(QMouseEvent *event)
-{
-	assert(event);
-}
-
 void Viewport::wheelEvent(QWheelEvent *event)
 {
 	assert(event);
--- pulseview-a7553da/pv/view/viewport.h~	2013-03-09 16:09:28.000000000 +0100
+++ pulseview-a7553da/pv/view/viewport.h	2013-03-09 17:23:07.126116283 +0100
@@ -48,7 +48,6 @@
 private:
 	void mousePressEvent(QMouseEvent *event);
 	void mouseMoveEvent(QMouseEvent *event);
-	void mouseReleaseEvent(QMouseEvent *event);
 	void wheelEvent(QWheelEvent *event);
 
 private:
--- pulseview-a7553da/test/CMakeLists.txt~	2013-03-09 16:09:28.000000000 +0100
+++ pulseview-a7553da/test/CMakeLists.txt	2013-03-09 17:27:36.806516037 +0100
@@ -35,6 +35,7 @@
 
 set(PULSEVIEW_LINK_LIBS
 	${Boost_LIBRARIES}
+	${CMAKE_THREAD_LIBS_INIT}
 )
 
 add_executable(pulseview-test
commit 4b963e7f126d2a13c4a4156fe1cfa8000c00fec0
Author: Joel Holdsworth <joel@airwebreathe.org.uk>
Date:   Sat Mar 9 16:52:04 2013 +0000

    Ensure sr_config_get is called even when assert is compiled out

diff --git a/pv/sigsession.cpp b/pv/sigsession.cpp
index a9da7bf..8f482f6 100644
--- a/pv/sigsession.cpp
+++ b/pv/sigsession.cpp
@@ -195,8 +195,10 @@ void SigSession::feed_in_header(const sr_dev_inst *sdi)
 
 	// Read out the sample rate
 	assert(sdi->driver);
-	assert(sr_config_get(sdi->driver, SR_CONF_SAMPLERATE,
-		(const void**)&sample_rate, sdi) == SR_OK);
+
+	const int ret = sr_config_get(sdi->driver, SR_CONF_SAMPLERATE,
+		(const void**)&sample_rate, sdi);
+	assert(ret == SR_OK);
 
 	// Create data containers for the coming data snapshots
 	{
